NEWS.md

```diff
@@ -11,6 +11,13 @@
 
 * Add read_***() functions with col_type parsers.
 
+# naomi 0.0.46
```

Comment:
The news item is for naomi v 0.0.46 but the current version in the Description is 0.0.49?

Can you bump the version number to 0.0.50 and set the news item as that?